syncd/syncd_mdio_ipc.cpp

```diff
@@ -0,0 +1,440 @@
+#include <stdio.h>
```

Comment:
if  you want to keep this file, make this to a proper class, and then instantiate object and call explicitly functions of that object